### PR TITLE
Fix cloudflare_split_tunnel import

### PR DESCRIPTION
**Pull request by tjstansell**
this should address #1313

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ BUG FIXES:
 * resource/cloudflare_argo: call `Read` for `Import` operations ([#1295](https://github.com/cloudflare/terraform-provider-cloudflare/issues/1295))
 * resource/cloudflare_argo_tunnel: update CNAME to use `cfargotunnel.com` ([#1293](https://github.com/cloudflare/terraform-provider-cloudflare/issues/1293))
 * resource/cloudflare_origin_ca_certificate: reintroduce `DiffSuppressFunc` for `requested_validity` changes to handle all schema/SDK combinations ([#1289](https://github.com/cloudflare/terraform-provider-cloudflare/issues/1289))
+* resource/cloudflare_split_tunnel: fix importing ([#1313](https://github.com/cloudflare/terraform-provider-cloudflare/issues/1313))
```
Comment (jacobbednarz) [inline CHANGELOG.md:21]: the CHANGLOG is automatically generated; please see [changelog-process](https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/docs/changelog-process.md) for details on populating this entry.

Comment (tjstansell) [reply CHANGELOG.md]: yep. i realized my mistake just after posting the PR... my bad.  should be fixed now.

File: website/docs/r/split_tunnel.html.markdown [hunk 2]
```diff
@@ -40,6 +41,7 @@ resource "cloudflare_split_tunnel" "example_split_tunnel_include" {
 The following arguments are supported:
 
 - `account_id` - (Required) The account to which the device posture rule should be added.
+- `mode` - (Required) The split tunnel mode.  Valid values are `include` or `exclude`.
```
Comment (jacobbednarz) [inline website/docs/r/split_tunnel.html.markdown:44]:
```suggestion
- `mode` - (Required) The split tunnel mode. Valid values are `include` or `exclude`.
```

File: website/docs/r/split_tunnel.html.markdown [hunk 3]
```diff
@@ -3,13 +3,14 @@ layout: "cloudflare"
...
-  Provides a Cloudflare Split Tunnel Exclude resource.
+  Provides a Cloudflare Split Tunnel resource.
 ---
 
 # cloudflare_split_tunnel
 
 Provides a Cloudflare Split Tunnel resource. Split tunnels are used to either
-include or exclude lists of routes from the WARP client's tunnel.
+include or exclude lists of routes from the WARP client's tunnel.  A
+single resource should be used choosing either the `include` or `exclude` mode.
```
Comment (jacobbednarz) [inline website/docs/r/split_tunnel.html.markdown:13]:
```suggestion
include or exclude lists of routes from the WARP client's tunnel.
```

**Comment by jacobbednarz** [comment]
thanks for this!

**Pull request merged**